cves/CVE-2013-0863.yml

```diff
@@ -236,8 +244,9 @@ subsystem:
     If this involves fixing multiple subsystems, you can make this an array.
 
     In 'answer', explain how you arrived at this determination.
-  answer:
-  name:
+  answer: This vulnerability caused a regression crash in avcodec. The fix was to correct the
```

Comment:
Good find, I'd also explain what a `regression crash` is. What regressed specifically? I'm assuming it's what you listed right after but making this more verbose would help readability a fair bit.